templates/README.md

```diff
@@ -33,4 +33,20 @@ clusterctl generate cluster ibm-powervs-1 --kubernetes-version v1.21.2 \
 --control-plane-machine-count=3 \
 --worker-machine-count=1 \
 --from ./cluster-template-powervs.yaml
-```
```

Comment:
will be good to add some information here - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/docs/book/src/topics/powervs/creating-a-cluster.md